assembly-multiuser/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/multiuser.properties

```diff
@@ -77,3 +77,7 @@ che.limits.organization.workspaces.count=-1
 #     additional workspace, they will be prompted with an error message. The
 #     organization will need to stop a running workspace to activate another.
 che.limits.organization.workspaces.run.count=-1
+#####                             CHE SYSTEM                                        #####
+#     Grant system permission for 'che.admin.name' user. If the user already exists it'll happen on
+#     component startup, if not - during the first login when user is persisted in the database.
+che.admin.name=admin
```

Comment:
What about renaming property to `che.system.admin_name`?https://github.com/eclipse/che/pull/6414#discussion_r140727000